README.md

```diff
@@ -1,41 +1,71 @@
-# webostv-as-promised
+# webos-as-promised
 
 Wrapper library [webOSTV.js](http://webostv.developer.lge.com/api/webostvjs/) on Promise
 
 ## Install
 
 ```bash
-npm i webostv-as-promised -S
+npm i webos-as-promised -S
 ```
 
 ## Usage
 
 ### webOS API
 
 ```javascript
-import { WebOSPromised } from 'webostv-as-promised';
+import { promisifyWebOS } from 'webos-as-promised';
 
-const webOS = new WebOSPromised;
+const promisedWebOS = promisifyWebOS(window.webOS);
 
-console.log(webOS.libVersion);
+console.log(promisedWebOS.libVersion);
 
-console.log(webOS.systemInfo());
+console.log(promisedWebOS.systemInfo());
 
-webOS.deviceInfo()
+promisedWebOS.deviceInfo()
   .then(info => console.log(info));
+
+const systemTimeRequest = promisedWebOS.service.request('luna://com.palm.systemservice', {
+    method: 'time/getSystemTime',
+    parameters: { subscribe: true }
+});
+systemTimeRequest.promise.then(res => console.log(res));
+// or
+systemTimeRequest.result.cancel();
 ```
 
 ### webOSDev API
 
 ```javascript
-import { WebOSDevPromised } from 'webostv-as-promised';
+import { promisifyWebOSDev } from 'webos-as-promised';
 
-const webOSDev = new WebOSDevPromised;
+const promisedWebOSDev = promisifyWebOSDev(window.webOSDev);
 
-console.log(webOSDev.APP.BROWSER);
+console.log(promisedWebOSDev.APP.BROWSER);
 
-webOSDev.LGUDID()
+promisedWebOSDev.LGUDID()
   .then(res => console.log(res.id));
 
-console.log(webOSDev.launchParams());
+console.log(promisedWebOSDev.launchParams());
+```
+
+## webOSTV.js library
+
+Folder `lib` contains a source code (`webOSTV.js`, `webOSTV-dev.js`) of the webOSTV.js library which may be includes to the js bundle for LG webOS web application.
```

Comment:
I think it's better to link webos.js download page http://webostv.developer.lge.com/api/webostvjs/
not keeping their sources inside the lib folder.